test/unit/helpers/rubygems_helper_test.rb

```diff
@@ -60,6 +60,7 @@ class RubygemsHelperTest < ActionView::TestCase
     version = build(:version)
     linkset = build(:linkset, docs: nil)
 
+    @virtual_path = "rubygems.show"
```

Comment:
I think that's because of [view template key lookup](https://github.com/rails/rails/blob/0fb2d1a0ba3360e928a7ac9d20378d7f5fbf915f/actionview/lib/action_view/helpers/translation_helper.rb#L116) is used in helpers..

```ruby
link_to t(".links.rss") #=> (looks for @virtual_path + key.to_s)
```

I think helper should not depend on `@virtual_path`. It will be better to use full key path for those string in those helpers and move them out of "view template" keys.